### PR TITLE
docs(ja): modify description about configuration-srcdir

### LINKED ISSUE #1421: docs(ja): modify sample description about configuration-srcdir page.
## About
Origin doc is as follows.
>Instead of example 1 you can also move the nuxt.config into your client folder. In this case you only need to specify client as the rootDir and you can leave srcDir empty:

https://nuxtjs.org/docs/2.x/configuration-glossary/configuration-srcdir

Japanese doc is as follows.
>例 1 の代わりに nuxt.config を src フォルダに移動することもできます。この場合 client を rootDir として指定するだけでよく、srcDir は空にできます:

## Suggesting
src フォルダに移動する → client フォルダに移動する

https://ja.nuxtjs.org/docs/2.x/configuration-glossary/configuration-srcdir

### PR DESCRIPTION
**Pull request by Bookman0001**
## What Done
resolved #1421.
resolved https://github.com/vuejs-jp/ja.nuxtjs.org/issues/298
Please check it!

### CODE REVIEW

File: content/ja/guides/configuration-glossary/configuration-srcdir.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ export default {
 
 例 2:
 
-例 1 の代わりに nuxt.config を `src` フォルダに移動することもできます。この場合 client を `rootDir` として指定するだけでよく、`srcDir` は空にできます:
+例 1 の代わりに nuxt.config を `client` フォルダに移動することもできます。この場合 client を `rootDir` として指定するだけでよく、`srcDir` は空にできます:
```
Comment (aytdm) [inline content/ja/guides/configuration-glossary/configuration-srcdir.md:50]:
```suggestion
例 1 の代わりに nuxt.config を `client` フォルダに移動することもできます。この場合 `client` を `rootDir` として指定するだけでよく、`srcDir` は空にできます:
```

Could you enclose "client" in backticks?

rel: https://github.com/nuxt/nuxtjs.org/commit/57bb31b50c322a4f58052c5aa3199d15ff44d152

**Review by aytdm — COMMENTED** [review]
Thank you :)

File: content/ja/guides/configuration-glossary/configuration-srcdir.md (hunk 1 above)
Comment (Bookman0001) [reply content/ja/guides/configuration-glossary/configuration-srcdir.md]: @aytdm Thank you for review! 

> Could you enclose "client" in backticks?

I fixed it in 9ff39ac86510c56cf658e9ed8fc27b60a5e0bcb3 .

**Review by aytdm — APPROVED** [review]
Thanks. First-time contribution :tada:

**Pull request merged**